examples/crd/crd-v1alpha2.yaml

```diff
@@ -9,3 +9,26 @@ spec:
     kind: TFJob
```

Comment:
Why are you doing this here and not in the ksonnet package?
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/tf-job-operator.libsonnet#L44